tests/e2e/tests/web/smoke/database/add-db-from-welcome-page.e2e.ts

```diff
@@ -39,10 +38,6 @@ test
     });
 
 test('Verify that all the links are valid from Welcome page', async t => {
-    // Verify build from source link
-    await t.click(welcomePage.buildFromSource);
```

Comment:
can you change this sourcePage verification to Linux?
we have a new link to Linux instead of source page

Reply:
sure,
will do